firecares/settings/base.py

```diff
@@ -157,6 +157,7 @@
 )
 
 INSTALLED_APPS = (
+    'tastypie',
```

Comment:
I'd put 'tastypie' lower in the stack, after the django built-ins

Reply:
:)

Reply:
👍